### PR TITLE
Late night changes 

### PR DESCRIPTION
**Pull request by maxwieringa**
Made a few changes to my code so it works perfectly fine, I also fixed the deprecation warning so that doesn't display in the terminal.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -47,11 +48,17 @@ app.use(express.static('static'));
...
+  hsts: false
+}));
 
 const db = mongoose.connection;
 
 // connect mongoose with the database
 // eslint-disable-next-line no-undef
+mongoose.set('useNewUrlParser', true);
+mongoose.set('useFindAndModify', false);
+mongoose.set('useCreateIndex', true);
```
Comment (pepijnkok) [inline app.js:54]: Nice fix for the deprecation warning

**Comment by daoneandonly** [comment]
Please merge the version of `develop` into this branch and see what is still applicable. If we're using helmet, make sure to check it functionality and make use of it. Just adding it for the sake of adding it will not do much to the project or our grading 😝 .

**Comment by maxwieringa** [comment]
> Please merge the version of `develop` into this branch and see what is still applicable. If we're using helmet, make sure to check it functionality and make use of it. Just adding it for the sake of adding it will not do much to the project or our grading 😝 .

Helmet is functional. You can check by looking at the network tab while inspecting the webpage.
Screenshot 2021-04-01 at 21 45 48
Helmet changes these HTTP headers for security reasons, if you're interested I have more info in the wiki :)

**Review by daoneandonly — APPROVED** [review]
Nice to have helmet in case of security issues.  👨‍💻

**Pull request merged**